code/datums/mutations/body.dm

```diff
@@ -394,3 +394,79 @@
+	text_gain_indication = "<span class='warning'>You get an intense feeling of heartburn.</span>"
+	text_lose_indication = "<span class'notice'>Your internal organs feel at ease.</span>"
+
+/datum/mutation/human/martyrdom/on_acquiring()
+	RegisterSignal(owner, COMSIG_MOB_STATCHANGE, .proc/bloody_shower)
+
+/datum/mutation/human/martyrdom/on_losing()
+	UnregisterSignal(owner, COMSIG_MOB_STATCHANGE)
+
+/datum/mutation/human/martyrdom/proc/bloody_shower(new_stat)
+	if(new_stat != UNCONSCIOUS)
+		return
+	var/list/organs = owner.getorganszone(BODY_ZONE_HEAD, 1)
+
+	for(var/obj/item/organ/I in organs)
+		I.Remove(owner, TRUE)
+
+	explosion(get_turf(owner), 0, 0, 2, 0, TRUE)
+	for(var/mob/living/carbon/human/H in range(2,owner))
+		var/obj/item/organ/eyes/eyes = H.getorganslot(ORGAN_SLOT_EYES)
+		to_chat(H, "<span class='userdanger'>You are blinded by a shower of blood!</span>")
+		H.Stun(20)
+		H.blur_eyes(20)
+		eyes?.applyOrganDamage(5)
+		H.confused += 3
+	for(var/mob/living/silicon/S in range(2,owner))
+		to_chat(S, "<span class='userdanger'>Your sensors are disabled by a shower of blood!</span>")
+		S.Paralyze(60)
+	owner.gib()
+
+/datum/mutation/human/headless
+	name = "H.A.R.S."
+	desc = "A mutation that makes the body reject the head. Stands for Head Allergic Rejection Syndrome. Warning: Removing this mutation is very dangerous, though it will regenerate head organs."
+	difficulty = 12 //pretty good for traitors
+	quality = NEGATIVE //holy shit no eyes or tongue or ears
+	text_gain_indication = "<span class='warning'>Something feels off.</span>" //heh
+
+/datum/mutation/human/headless/on_acquiring()
+	. = ..()
+	if(.)//cant add
+		return TRUE
+	var/obj/item/organ/brain/brain = owner.getorganslot(ORGAN_SLOT_BRAIN)
+	if(brain) //so this doesn't instantly kill you
+		brain.organ_flags &= ~ORGAN_VITAL
+		qdel(brain)
+
+	var/obj/item/bodypart/head/head = owner.get_bodypart(BODY_ZONE_HEAD)
+	if(head)
+		owner.visible_message("<span class='warning'>[owner]'s head splatters with a sickening crunch!</span>", ignored_mobs = list(owner))
+		new /obj/effect/gibspawner/generic(get_turf(owner), owner)
+		head.dismember(BRUTE)
+		head.drop_organs()
+		qdel(head)
+		owner.regenerate_icons()
+	RegisterSignal(owner, COMSIG_LIVING_REGENERATE_LIMBS, .proc/rejecthead)
+
+/datum/mutation/human/headless/on_losing()
+	. = ..()
+	var/successful = owner.regenerate_limb(BODY_ZONE_HEAD, noheal = TRUE) //noheal needs to be TRUE to prevent weird adding and removing mutation healing
+	if(!successful)
+		stack_trace("HADS mutation head regeneration failed! (usually caused by headless syndrome having a head)")
+		return TRUE // trouble
+	UnregisterSignal(owner, COMSIG_LIVING_REGENERATE_LIMBS)
+	owner.dna.species.regenerate_organs(owner, excluded_limbs = list(BODY_ZONE_CHEST)) //only regenerate head
+	owner.apply_damage(damage = 50, damagetype = BRUTE, def_zone = BODY_ZONE_HEAD) //and this to really prevent it
```

Comment:
Why is adding this brute damage necessary? If someone uses adding and curing this mutation to heal someone's head's brute and burn damage, I say that that's a really metal thing that should be kept around.

Reply:
it's to discourage using it to create organs, though that is possible with some healing around